File: pm/src/main/java/org/sqlbuilder/pm/PbWordResolver.java
package org.sqlbuilder.pm;

import org.sqlbuilder.common.DeSerialize;
import org.sqlbuilder.common.Resolver;

import java.io.File;
import java.io.IOException;

public class PbWordResolver extends Resolver<String,Integer>
{
	public PbWordResolver(final String ser) throws IOException, ClassNotFoundException
	{
		super(DeSerialize.deserialize(new File(ser)));
	}
}

File: pm/src/main/java/org/sqlbuilder/pm/PbWordResolver.kt
package org.sqlbuilder.pm

import org.sqlbuilder.common.DeSerialize.deserialize
import org.sqlbuilder.common.Resolver
import java.io.File

typealias PmPbWordResolvable = String
typealias PmPbWordResolved = Int

class PbWordResolver(ser: String) : Resolver<PmPbWordResolvable, PmPbWordResolved>(deserialize<Map<PmPbWordResolvable, PmPbWordResolved>>(File(ser)))
